Improve Ballerina quick-tour (#1357)
ballerina-platform/ballerina-www: website/two-column-pages/docs/learn/quick-tour.md

string status = check request.getTextPayload();
```

> And then change the signature of `sayHello` function to add `returns error?` as `check` will return if error value is encountered as the return value of  `request.getTextPayload()`.

Reviewer: ```suggestion
Change the signature of the `sayHello` resource function to add `returns error?`, so that `check` will return the error value if `request.getTextPayload()` evaluates to `error`.
```

> > **Tip**: The check keyword means that this may return an error but I do not want to handle it here - pass it further away (to the caller function, or if this is a top-level function - generate a runtime failure).

Reviewer: ```suggestion
> **Tip**: The 'check' keyword means that the returned error may not be handled within the same function.
Instead, it may be passed to the caller function. If it is a top-level function, then runtime failure will be generated.
```

> > **Tip**: The `check` keyword means that if the expression evaluates to an `error`, the returned error is not handled within the same function. For instance this `resource function sayHello` will send a '500 Internal Server Error' as it's response and will exit the function at the point of `check` given the expression evaluates to `error`.

Reviewer 2: How about 
```suggestion
> **Tip**: The `check` keyword means that if the expression evaluates to an `error`, the returned error is not handled within the same function. For instance with the `resource function sayHello`, the function execution would stop and "500 Internal Server Error" would be set as the response, if the given expression evaluates to `error` where `check` is used.
```

> > **Tip**: The `check` keyword means that if the expression evaluates to an `error`, the returned error is not handled within the same function. For instance with the `resource function sayHello`, the function execution would stop and "500 Internal Server Error" would be set as the response, if the given expression evaluates to `error` where `check` is used.

Reviewer: The `check` keyword denotes that if the expression evaluates to `error`, then the returned error is not handled within the same function. For example, when`check` is used with the `sayHello` resource function and if the given expression evaluates to `error`, the function execution would stop and "500 Internal Server Error" would be set as the response.

string status = check request.getTextPayload();
```

Change the signature of the `sayHello` resource function to add `returns error?`, so that `check` will return the error value if `request.getTextPayload()` evaluates to `error`.

> > **Tip**: The 'check' keyword means that the returned error may not be handled within the same function.

Reviewer 2: ```suggestion
> **Tip**: The `check` keyword means that if the expression evaluates to an `error`, the returned error is not handled within the same function.
```

Not sure if "may" is correct here? Given that the error will be returned at this point for sure?

> Instead, it may be passed to the caller function. If it is a top-level function, then runtime failure will be generated.

Reviewer 2: Is "If it is a top-level function, then runtime failure will be generated." correct? Since the error is returned to the callee function?

Author: Here the top-level means a bottom of the stack, main or a resource function, hence there is no caller to return to.

Reviewer 2: Not sure if run time failure is the correct term? Currently with a main, there is no failure if an error is returned. Even for services, the service still runs right?

}
@docker:Expose {}
listener http:Listener cmdListener = new(9090);

@http:ServiceConfig {
   basePath: "/"
}
service hello on cmdListener {
```

> > **Note**: On Windows, make sure Docker runs with Linux containers and in the general settings, enable `Expose daemon on tcp://localhost:2375 without TLS`. For more details, see the [Docker README](https://github.com/ballerinax/docker/blob/master/samples/README.md).

Reviewer: ```suggestion
> **Note**: On Windows, make sure Docker runs with Linux containers, and in the general settings, enable `Expose daemon on tcp://localhost:2375 without TLS`. For more details, see the [Docker README](https://github.com/ballerinax/docker/blob/master/samples/README.md).
```
